pyproject.toml

```diff
@@ -165,13 +169,29 @@ enable_error_code = ["ignore-without-code", "redundant-expr", "truthy-bool"]
 warn_unreachable = true
 disallow_untyped_defs = false
 disallow_incomplete_defs = false
+# data-apis/array-api#589
+disallow_any_expr = false
 
 [[tool.mypy.overrides]]
 module = "array_api_extra.*"
 disallow_untyped_defs = true
 disallow_incomplete_defs = true
 
 
+# pyright
+
+[tool.pyright]
+include = ["src", "tests"]
+pythonVersion = "3.10"
+pythonPlatform = "All"
+typeCheckingMode = "strict"
+
+# data-apis/array-api#589
+reportAny = false
+reportExplicitAny = false
+reportUnknownMemberType = false
```

Comment:
Instead of disabling this globally, you could also consider using the `baseline` feature:
https://docs.basedpyright.com/latest/benefits-over-pyright/baseline/

Reply:
> I was mostly talking about the reportUnknownMemberType

These are all about the types coming from array-api-strict I think:

`/array-api-extra/tests/test_funcs.py:183:13 - error: Type of "ones" is partially unknown
    Type of "ones" is "(shape: int | Tuple[int, ...], *, dtype: _DType | None = None, device: Unknown | None = None) -> Array" (reportUnknownMemberType)`

If so, I think more of those will continue to be introduced for the time being, so I'd rather ignore globally.

Reply:
But even so, it's probably best to baseline those errors, than to globally hide errors about unknown member types, IMO

Reply:
Alternatively, you could als try to create some quick 'n dirty stubs for it

Reply:
I'll leave it for this PR and open an issue 👍